include/taskolib/Step.h

```diff
@@ -315,6 +315,9 @@ class Step
      * given Context.
      */
     void copy_used_variables_from_lua_to_context(const sol::state& lua, Context& context);
+
+    /// Execute the Lua script, throwing an exception if anything goes wrong.
```

Comment:
Do you use the different comment style (`///` vs `/*`) depending on number of lines?

Whatever, maybe it would be helpful to specify what is returned here. But then, it's internal, so maybe ignore.

Reply:
Yes, I use `///` for one-line Doxygen comments and
```
/**
 *
 */
```
for multi-liners.

Maybe the comment should have a `\see execute()` or something like this. Will change it.

Reply:
Done.